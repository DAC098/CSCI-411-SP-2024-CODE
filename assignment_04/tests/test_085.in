7
-1 -1 2
tgtctgt ataaaataat
rbpxxwlljajfmdxtlpiekyafuyfmyj lymnpekpgphljo
gtwmcpaksymuhrfdjbjjsawgkscddcqo rmanidlarzztofb
aagaagcaataaaaa aaaatc
uiwbdbohkvjuvgrbhbzbjsklibmhemartx xjwciewwgokkufpl
xyxikxuogvohwfznhvzhapbblgbjbotqkjhcob cfksczandvebmyobfiktdqqxtmdnlkmkqxahfom
aacagattaaaa aaagaggaaaaacgaaaga
