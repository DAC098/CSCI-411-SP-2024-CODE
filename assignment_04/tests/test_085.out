__________tgtctgt
ataaaataat_______
-17
______________rbpxxwlljajfmdxtlpiekyafuyfmyj
lymnpekpgphljo______________________________
-44
_______________gtwmcpaksymuhrfdjbjjsawgkscddcqo
rmanidlarzztofb________________________________
-47
______aagaagcaataaaaa
aaaatc_______________
-21
________________uiwbdbohkvjuvgrbhbzbjsklibmhemartx
xjwciewwgokkufpl__________________________________
-50
_______________________________________xyxikxuogvohwfznhvzhapbblgbjbotqkjhcob
cfksczandvebmyobfiktdqqxtmdnlkmkqxahfom______________________________________
-77
___________________aacagattaaaa
aaagaggaaaaacgaaaga____________
-31
